### PR TITLE
add second logo for navbar

### PR DESCRIPTION
**Pull request by dyang108**

### CODE REVIEW

**Comment by dyang108** [comment]
next up: more appealing nav items

**Comment by dyang108** [comment]
[
Screen Shot 2021-01-18 at 3 25 13 PM
](url)

File: src/pages/Main/Main.css [hunk 1]
```diff
@@ -1,6 +1,15 @@
 .Main__hero {
   display: flex;
-  align-items: center;
-  justify-content: center;
+  /* align-items: center; */
+  /* justify-content: center; */
```
Comment (amarcher) [inline src/pages/Main/Main.css]:
```suggestion
```

File: src/pages/Main/index.tsx [hunk 2]
```diff
@@ -30,24 +30,19 @@ export default function Main() {
...
   return (
     <>
-      <Title>{APP_NAME}</Title>
-
-      <div className="Main__hero">
-        <Logo />
-      </div>
-
-      {data?.categories && <p>Choose a category:</p>}
+      <ul>
```
Comment (amarcher) [inline src/pages/Main/index.tsx:35]: Nice

File: src/index.css [hunk 3]
```diff
@@ -6,11 +6,18 @@ body {
...
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: -apple-system, BlinkMacSystemFont, 'Trebuchet MS', 'Segoe UI',
+    'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans',
+    'Helvetica Neue', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: linear-gradient(to bottom right, #fff, #e2cc9e);
+  background-size: auto 100%;
```
Comment (amarcher) [inline src/index.css:15]: Necessary?

File: src/components/Logo2.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
...22 58.2632C218.956 56.8667 217.2 55.8087 215.254 55.0893C213.349 54.3276 211.466 53.9467 209.604 53.9467ZM278.667 47.4721C281.121 47.4721 283.174 47.9799 284.824 48.9956C286.517 50.0112 287.871 51.3654 288.886 53.0581C289.944 54.7508 290.685 56.7186 291.108 58.9614C291.574 61.1619 291.806 63.4471 291.806 65.8169C291.806 68.1443 291.679 70.9585 291.426 74.2592C291.172 77.5177 290.621 80.8397 289.775 84.2251L281.968 83.0825C282.475 80.0779 282.814 77.3273 282.983 74.8305C283.152 72.2915 283.237 70.091 283.237 68.229C283.237 63.701 282.75 60.5272 281.777 58.7075C280.846 56.8455 279.386 55.9145 277.397 55.9145C275.662 55.9145 273.779 56.7186 271.748 58.3266C269.717 59.9347 267.728 62.368 265.781 65.6264C263.834 68.8849 262.057 72.9897 260.449 77.9409C258.841 82.8921 257.593 88.7107 256.704 95.3969V117.804L247.119 117.296C247.077 104.982 246.95 94.466 246.738 85.7485C246.569 76.9887 246.399 69.837 246.23 64.2934C245.976 57.7765 245.701 52.4233 245.405 48.2339L256.704 47.4721V68.229C257.55 65.9438 258.629 63.5952 259.941 61.1831C261.295 58.7286 262.882 56.4858 264.702 54.4546C266.564 52.4233 268.637 50.7518 270.923 49.4399C273.25 48.1281 275.831 47.4721 278.667 47.4721Z" fill="#0A222A"/>
```
Comment (amarcher) [inline src/components/Logo2.tsx]: Likely needs eslint fix here

File: src/components/Nav.css [hunk 5]
```diff
@@ -2,23 +2,26 @@
   position: -webkit-sticky;
   position: sticky;
   top: 0;
+  left: 10%;
+  right: 10%;
```
Comment (amarcher) [inline src/components/Nav.css]: Might need to revisit these %-widths (they prop ought to pixels)

File: src/components/Nav.css [hunk 6]
```diff
@@ -2,23 +2,26 @@
...
   z-index: 10;
 }
 
 .nav__content {
-  height: 40px;
+  height: 50px;
   display: flex;
-  justify-content: center;
+  justify-content: left;
```
Comment (amarcher) [inline src/components/Nav.css]: This value is invalid
```suggestion
  justify-content: flex-start;
```

File: src/components/Nav.tsx [hunk 7]
```diff
@@ -89,6 +90,11 @@ export default function Nav() {
     <div className="nav">
       <nav className="nav__content">
         <ul className="nav__list">
+          <li className="nav__list-item" style={{ paddingTop: '7px' }}>
+            <a href="/">
+              <Logo></Logo>
```
Comment (amarcher) [inline src/components/Nav.tsx]:
```suggestion
              <Logo />
```

File: src/pages/Main/index.tsx [hunk 8]
```diff
@@ -31,27 +31,20 @@ export default function Main() {
...
-              <Button href={`/category/${id}`}>
-                {name} ({projects?.length})
-              </Button>
-            </p>
-          );
-        })}
-
+      <ul>
+        {data?.categories &&
+          data.categories.map((category) => {
```
Comment (amarcher) [inline src/pages/Main/index.tsx]:
```suggestion
        {data?.categories?.map((category) => {
```

File: src/pages/Main/index.tsx [hunk 9]
```diff
@@ -31,27 +31,19 @@ export default function Main() {
...
-              <Button href={`/category/${id}`}>
-                {name} ({projects?.length})
-              </Button>
-            </p>
-          );
-        })}
-
+      <ul>
+        {data?.categories?.map((category) => {
+            const { name, id, projects } = category || {};
```
Comment (amarcher) [inline src/pages/Main/index.tsx:37]:
```suggestion
        {data?.categories?.map({ name, id, projects } = {}) => {
```

Comment (amarcher) [reply src/pages/Main/index.tsx:37]: NM, could be nul

**Review by amarcher — APPROVED** [review]

File: src/components/Nav.css (hunk 5 above)
Comment (amarcher) [inline src/components/Nav.css]:
```suggestion
```

**Review by amarcher — APPROVED** [review]

**Pull request merged**